src/go/rpk/pkg/cli/cmd/debug/bundle.go

```diff
@@ -24,11 +24,15 @@ import (
 )
 
 // Use the same date specs as journalctl (see `man journalctl`).
-const timeHelpText = `(journalctl date format, e.g. YYYY-MM-DD)`
+const (
+	timeHelpText = `(journalctl date format, e.g. YYYY-MM-DD)`
+	outputFlag   = "output"
```

Comment:
Why extract `output` to a dedicated const, vs using it in the one place it's used below?

Reply:
It's used in 2 parts: L63 and L101